pkg/api/v1beta2/dynakube/dynakube_types.go

```diff
@@ -59,9 +57,11 @@ type DynaKubeValueSource struct { //nolint:revive
 }
 
 // +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+// +kubebuilder:storageversion
```

Comment:
is `+kubebuilder:storageversion` necessary twice?

Reply:
missed it, nice catch 🍪